goosebit.yaml

```diff
@@ -1,32 +1,46 @@
 ## Settings to adjust for each installation
+## Settings are listed with single #, comments with 2 #
```

Comment:
All these unrelated changes should go into a separate commit. Anyway, not a fan, sorry.